client/flutter/pubspec.yaml

```diff
@@ -75,9 +75,7 @@ flutter:
     - assets/images/
     - assets/svg/
     - assets/animations/
-    - assets/news_press/
     - assets/onboarding/
-    - assets/travel_advice/
```

Comment:
These files are referenced in the arb file file, where they aren't removed. I'm expecting for them to be removed from the arb file(s) or to explain why it should be done any other way.

https://github.com/WorldHealthOrganization/app/blob/c02ffef9a0151dbe8248dcc745efa35b859b16b9/client/flutter/lib/l10n/intl_en_US.arb#L68

Reply:
fixed